code/modules/client/preferences/body_size.dm

```diff
@@ -0,0 +1,16 @@
+/datum/preference/choiced/body_size
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	savefile_identifier = PREFERENCE_CHARACTER
+	priority = PREFERENCE_PRIORITY_BODY_TYPE
+	savefile_key = "body_size"
+	savefile_identifier = PREFERENCE_CHARACTER
```

Comment:
Tweaks here:
(`priority` isn't needed since none of the other preferences are messing with the body size, and `savefile_identifier` is being defined twice.)
```suggestion
/datum/preference/choiced/body_size
	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
	savefile_identifier = PREFERENCE_CHARACTER
	savefile_key = "body_size"
```